Assets/HoloToolkit/ARCA/Scripts/SpatialSync/MarkerDetectionHololens.cs

```diff
@@ -15,33 +17,113 @@ public class MarkerDetectionHololens : MonoBehaviour
 		// rot : the marker rotation
 		public delegate void OnMarkerDetectedDelegate(int markerId, Vector3 pos, Quaternion rot);
 		public OnMarkerDetectedDelegate OnMarkerDetected;
+
+	    /// <summary>
+	    /// A component for capturing photos from the HoloLens webcam
+	    /// </summary>
 		[Tooltip("A component for capturing photos from the HoloLens webcam")]
-		public CameraCaptureHololens HoloLensCapture;
+	    [SerializeField]
+		private CameraCaptureHololens holoLensCapture;
+
+	    /// <summary>
+	    /// The physical size of the marker to search for in metres
+	    /// </summary>
 		[Tooltip("The physical size of the marker to search for in metres")]
-		public float MarkerSize = 0.05f;
+	    [SerializeField]
```

Comment:
strange formatting

Reply:
It seems fine to me, what are you referring specifically?